### PR TITLE
[IMP] mail: remove the delete button

### PR DESCRIPTION
**Pull request by nihp-odoo**
Remove the custom delete button from email templates' form view.
And a tiny UI/UX change in delete confirmation dialog.

Task-4231737

### CODE REVIEW

File: addons/mail/static/src/core/web/mail_composer_template_selector.js [hunk 1]
```diff
@@ -51,7 +51,8 @@ export class MailComposerTemplateSelector extends Component {
      */
     async onDeleteTemplate(template) {
         this.env.services.dialog.add(ConfirmationDialog, {
-            body: sprintf(_t('Are you sure you want to delete "%(template_name)s"?'), {
+            title: ("Delete Email Template"),
```
Comment (tde-banana-odoo) [inline addons/mail/static/src/core/web/mail_composer_template_selector.js:54]: :socks: alphabetical ordering

**Review by tde-banana-odoo — COMMENTED** [review]
Small linting remark, otherwise LGTM. Could you use the REF or IMP tag ? REM is more for big removals

File: addons/mail/static/src/core/web/mail_composer_template_selector.js [hunk 2]
```diff
@@ -61,7 +61,9 @@ export class MailComposerTemplateSelector extends Component {
                     return current.id !== template.id;
                 });
             },
+            cancelLabel: ("No, keep it"),
```
Comment (tde-banana-odoo) [inline addons/mail/static/src/core/web/mail_composer_template_selector.js:64]: Isn't the translate tool missing ?

File: addons/mail/static/src/core/web/mail_composer_template_selector.js [hunk 3]
```diff
@@ -61,7 +61,9 @@ export class MailComposerTemplateSelector extends Component {
...
             cancel: () => {},
+            title: ("Delete Email Template"),
```
Comment (tde-banana-odoo) [inline addons/mail/static/src/core/web/mail_composer_template_selector.js:66]: Isn't the translate tool missing ?

File: addons/mail/static/src/core/web/mail_composer_template_selector.js (hunk 2 above)
Comment (nihp-odoo) [reply addons/mail/static/src/core/web/mail_composer_template_selector.js]: Hello again @tde-banana-odoo 

Done! just waiting for the runbot :hourglass_flowing_sand: 
I was confused about not adding translate tool in master, but we just don't add pot :sweat_smile:.

Thanks :smile:

**Comment by tde-banana-odoo** [comment]
@robodoo r+

**Pull request closed without merging**